File: belajar-loop/Belajar-loop2/loop2.js

/*
 * Level4

 1
 23
 456
 78910

 *Level 5

    *
   ***
  *****
 *******

 */

let a = 4;
let b = 1;

for(let i=0;i<a;i++){
	for(let j=0;j<=i;j++){
		process.stdout.write(b.toString());
		b++;
	}
	console.log();
}
console.log();
for(let i=0;i<a*2;i+=2){
	for(let j=0;j<=a-i/2;j++){
		process.stdout.write(" ");
	}
	for(let j=0;j<=i;j++){
		process.stdout.write("*");
	}
	console.log();
}

File: belajar-loop/Belajar-loop2/loop2.lua
--[[

Level4

 1
 23
 456
 78910

 Level 5

    *
   ***
  *****
 *******

]]

local a = 4
local b = 1
for i=0,a-1 do
	for j=0,i do
		io.write(b)
		b=b+1
	end
	print()	
end
print()
for i=0,a*2-1,2 do
	for j=0,a-i/2+1 do
		io.write(" ")
	end
	for j=0,i do
		io.write("*")
	end
	print()
end
